styleguide.config.js

```diff
@@ -19,6 +19,9 @@ if (env.parsed && !env.error) {
 }
 
 module.exports = {
+    moduleAliases: {
+        'react-rainbow-components': path.resolve(__dirname, './src'),
```

Comment:
I think we should use:
```path.resolve(__dirname, './src/components'),```
since in the examples right now you are using:
```import { Accordion } from 'react-rainbow-components/components';```
and this is confuse since the real usage is:
```import { Accordion } from 'react-rainbow-components';```